tide-log/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+authors = [
+    "Tide Developers"
+]
+description = "Tide middleware for logging"
+documentation = "https://docs.rs/tide"
```

Comment:
When this crate is published, you might want to change the URL here :sweat_smile: 

Otherwise, looks good I think!

Reply:
Done. We also might want to add version to `tide` before release for all subcrates. I'll add them to #162